compiler/optimizer/OMRSimplifierTableEnum.hpp

```diff
@@ -149,6 +149,14 @@
    lxorSimplifier,          // TR::lxor
    bxorSimplifier,          // TR::bxor
    sxorSimplifier,          // TR::sxor
+   fmaSimplifier,           // TR::fmuladd
```

Comment:
The consistent naming is to use the opcode name here, so I'd like all the `ma`s spelled out